src/collections/_documentation/platforms/react-native/touchevents.md

```diff
@@ -0,0 +1,106 @@
+---
+title: 'Tracking Touch Events'
```

Comment:
what's about being a Breadcrumbs page instead of Touch events? I believe we'll be adding more of those and they would make sense to be aggregated IMO

Reply:
Hmm..yes it would make sense when we add more docs on breadcrumbs, but having this alone in there right now would be weird I'd think.